### ISSUE TITLE
Better keyboard shortcut

### ISSUE THREAD

**Issue by AntoineRobert** [roldanjrCodeArts9711/productivity-timer#74]
**Is your feature request related to a problem? Please describe.**
Currently the shortcuts are conflicting with the default eclipse shortcuts. (Alt + Shift + S)

**Describe the solution you'd like**
A way to disable/activate shortcuts.

**Describe alternatives you've considered**
A way to customize shortcut.

Thanks

**Comment by roldanjrCodeArts9711** [comment]
Thanks @AntoineRobert . I will try to add feature that enable changing the default shortcuts as soon as possible. 👍

**Comment by roldanjrCodeArts9711** [comment]
@AntoineRobert I'm sorry to tell. As of now I can't develop the feature you've requested. I'm really busy right now with new projects. 

If it is okay for you I will add it to app's `ROAD MAP` and develop it soon. Please let me know your thoughts. 

I've released new version of the app that includes new features and bug fixes. Suddenly the feature you've requested not included. Please see [Release Notes](https://github.com/roldanjrCodeArts9711/productivity-timer/releases/tag/v2.4.0-beta) I've created for more details of changes I've made.

I hope it would help you.

**Comment by cvolant** [comment]
That would be great indeed : I defined custom shortcuts in another app, and Productivity Timer (which is great beside that) break them by defining the same shortcuts as global.

**Comment by roldanjr** [comment]
Yes that's true @cvolant . But as of now I am busy building other things. 

The behavior I want to develop is kinda similar to some popular apps like Visual Studio Code.

![customizable-shortcut](https://user-images.githubusercontent.com/38913069/89181936-37d2ee00-d5c7-11ea-9799-47a82ef4d2ac.gif)


Developing this kind of feature will cause me a lot of time.
So I preferred adding it to app's road map.

If anyone would love to help me with this feature I will be very glad.
